HLTrigger/Configuration/python/HLT_75e33/modules/hltHighPtTripletStepTrackingRegionFromTrimmedVertices_cfi.py

```diff
@@ -0,0 +1,30 @@
+import FWCore.ParameterSet.Config as cms
+
+hltTrackingRegionFromTrimmedVertices = cms.EDProducer('GlobalTrackingRegionWithVerticesEDProducer',
+  RegionPSet = cms.PSet(
+    ptMin                   = cms.double(0.9),
+    beamSpot                = cms.InputTag('hltOnlineBeamSpot'),
+    VertexCollection        = cms.InputTag('hltPhase2TrimmedPixelVertices'),
+    originRadius            = cms.double(0.02),
+    precise                 = cms.bool(True),
+    useMultipleScattering   = cms.bool(False),
+    useFixedError           = cms.bool(True),
+    sigmaZVertex            = cms.double(3.0),
+    fixedError              = cms.double(0.2),
+    useFoundVertices        = cms.bool(True),
+    useFakeVertices         = cms.bool(False),
+    maxNVertices            = cms.int32(-1),
+    nSigmaZ                 = cms.double(4.0),
+    pixelClustersForScaling = cms.InputTag('siPixelClusters'),
```

Comment:
this is an offline product?

Reply:
yes, I copied it from the defaults. It is not used in this configuration, however I changed it to hltSiPixelClusters in d17f0fbff38c1a8589f544e295f42de5d6736a73